cpp/src/barretenberg/honk/composer/turbo_plonk_composer.hpp

```diff
@@ -0,0 +1,213 @@
+    }
+
+    void create_range_constraint(const uint32_t variable_index,
+                                 const size_t num_bits,
+                                 std::string const& msg = "create_range_constraint")
+    {
+        circuit_constructor.create_range_constraint(variable_index, num_bits, msg);
+    }
+
+    accumulator_triple create_logic_constraint(const uint32_t a,
+                                               const uint32_t b,
+                                               const size_t num_bits,
+                                               bool is_xor_gate)
+    {
+        return circuit_constructor.create_logic_constraint(a, b, num_bits, is_xor_gate);
+    }
+
+    accumulator_triple create_and_constraint(const uint32_t a, const uint32_t b, const size_t num_bits)
+    {
+        return circuit_constructor.create_and_constraint(a, b, num_bits);
+    }
+
+    accumulator_triple create_xor_constraint(const uint32_t a, const uint32_t b, const size_t num_bits)
+    {
+        return circuit_constructor.create_xor_constraint(a, b, num_bits);
+    }
+    uint32_t add_variable(const barretenberg::fr& in) { return circuit_constructor.add_variable(in); }
+
+    uint32_t add_public_variable(const barretenberg::fr& in) { return circuit_constructor.add_public_variable(in); }
+
+    virtual void set_public_input(const uint32_t witness_index)
+    {
+        return circuit_constructor.set_public_input(witness_index);
+    }
+
+    uint32_t put_constant_variable(const barretenberg::fr& variable)
+    {
+        return circuit_constructor.put_constant_variable(variable);
+    }
+
+    size_t get_num_constant_gates() const { return circuit_constructor.get_num_constant_gates(); }
+
+    bool check_circuit() { return circuit_constructor.check_circuit(); }
+
+    barretenberg::fr get_variable(const uint32_t index) const { return circuit_constructor.get_variable(index); }
+    /**Proof and verification-related methods*/
+
+    std::shared_ptr<bonk::proving_key> compute_proving_key()
+    {
+        return composer_helper.compute_proving_key(circuit_constructor);
+    }
+
+    std::shared_ptr<bonk::verification_key> compute_verification_key()
+    {
+        return composer_helper.compute_verification_key(circuit_constructor);
+    }
+
+    uint32_t zero_idx = 0;
+
+    void compute_witness() { composer_helper.compute_witness(circuit_constructor); };
+    // TODO(#230)(Cody): This will not be needed, but maybe something is required for ComposerHelper to be generic?
+    plonk::TurboVerifier create_verifier() { return composer_helper.create_verifier(circuit_constructor); }
+    /**
+     * Preprocess the circuit. Delegates to create_prover.
+     *
+     * @return A new initialized prover.
+     */
+    /**
+     * Preprocess the circuit. Delegates to create_unrolled_prover.
+     *
+     * @return A new initialized prover.
+     */
+    plonk::TurboProver preprocess() { return composer_helper.create_prover(circuit_constructor); };
```

Comment:
I think we can delete `preprocess()` here (and in the `standard_plonk_composer` if you'd like)

Reply:
Deleted all instances of `preprocess`